doc/API.md

```diff
@@ -27,13 +27,114 @@ Example GET curl call:  `curl -A "Sia-Agent" /wallet/transactions?startheight=1&
 
 Example POST curl call: `curl -A "Sia-Agent" --data "amount=123&destination=abcd" /wallet/siacoins
 
+Daemon
+------
+
+Queries:
+
+* /daemon/constants [GET]
+* /daemon/stop      [GET]
+* /daemon/version   [GET]
+
+#### /daemon/constants [GET]
```

Comment:
I was wondering, would it be useful to have an Advanced Overview plugin that just displayed lots of data in JSON format from various information/status calls like this? It'd be a pretty easy thing to make